thunderbird/MozillaThunderbird.spec

```diff
@@ -266,7 +264,7 @@ fi
 %patch8 -p1
 %patch9 -p1
 %patch10 -p1
-%ifarch s390x ppc64
+%ifarch s390x
```

Comment:
That patch is using `icudt${version}b.dat` instead of the `l.dat`-version. Shouldn't that also be applied for other big endian platforms?

Reply:
true, so is the list with s390x and ppc64 complete? That are not all BE ones, right?

Reply:
also in that case we should rename the patch as well at some point but wouldn't dare right now

Reply:
Those 2 are the ones I know of :-) Its probably not complete, no. And yes, I was thinking the same about renaming the patch. Might be best to just open an issue to do this in the future